azure-cognitiveservices-vision-computervision/README.md

```diff
@@ -0,0 +1,327 @@
+
+You can use Computer Vision in your application to:
+
+- Analyze images for insight
+- Extract text from images
+- Generate thumbnails
+
+Looking for source code or API reference?
+
+* [Pip package][pypi_computervision]
+* [SDK reference documentation][ref_computervision_sdk]
+* [SDK source code][source_code]
+
+## Prerequisites
+
+* Azure subscription - [Create a free account][azure_sub]
+* Azure [Computer Vision resource][computervision_resource]
+* [Python 3.6+][python]
+
+If you need a Computer Vision API account, you can create one with this [Azure CLI][azure_cli] command:
+
+```Bash
+RES_REGION=westeurope 
+RES_GROUP=<resourcegroup-name>
+ACCT_NAME=<computervision-account-name>
+
+az cognitiveservices account create \
+    --resource-group $RES_GROUP \
+    --name $ACCT_NAME \
+    --location $RES_REGION \
+    --kind ComputerVision \
+    --sku S1 \
+    --yes
+```
+
+## Installation
+
+Install the Azure Cognitive Services Computer Vision SDK with [pip][pip], optionally within a [virtual environment][venv].
+
+### Configure a virtual environment (optional)
+
+Although not required, you can keep your base system and Azure SDK environments isolated from one another if you use a [virtual environment][virtualenv]. Execute the following commands to configure and then enter a virtual environment with [venv][venv], such as `cogsrv-vision-env`:
+
+```Bash
+python3 -m venv cogsrv-vision-env
+source cogsrv-vision-env/bin/activate
+```
+
+### Install the SDK
+
+Install the Azure Cognitive Services Computer Vision SDK for Python with [pip][pip]:
+
+```Bash
+pip install azure-cognitiveservices-vision-computervision
+```
+
+## Authentication
+
+Once you create your Computer Vision resource, you need its **region**, and one of its **account keys** to instantiate the client object.
+
+Use these values when you create the instance of the [ComputerVisionAPI][ref_computervisionclient] client object. 
+
+### Get credentials
+
+Use the [Azure CLI][cloud_shell] snippet below to populate two environment variables with the Computer Vision account **region** and one of its **keys** (you can also find these values in the [Azure portal][azure_portal]). The snippet is formatted for the Bash shell.
+
+```Bash
+RES_GROUP=<resourcegroup-name>
+ACCT_NAME=<computervision-account-name>
+
+export ACCOUNT_REGION=$(az cognitiveservices account show \
+    --resource-group $RES_GROUP \
+    --name $ACCT_NAME \
+    --query location \
+    --output tsv)
+
+export ACCOUNT_KEY=$(az cognitiveservices account keys list \
+    --name $ACCT_NAME \
```

Comment:
Looks like this dupe `--name $ACCT_NAME \` argument is still in place in this second export command.

Reply:
fixed